### DISCUSSION TITLE
Review

### DISCUSSION THREAD

**Discussion by hadoan149** [lab-brussels-1/group5#12] [Review]
Something needs reviewing? Feel free to put it here 😃

**Comment by sammou00** [comment]
Well done so far , I was expecting your user-personas is ready by now but it is OK.

**Reply by hadoan149** [reply]
Hi Samir, yes it was added but not merged yet. Now it is 😊

**Comment by sammou00** [comment]
![image](https://user-images.githubusercontent.com/67711948/159570349-35bc9cc2-e9f3-4132-83a1-2db02b194f8f.png)

**Comment by sammou00** [comment]
- I week is a short time , yes but the project also is small and it doesn't need to be perfect because we are learning now.

- Members are beginners to coding ? yes, but you don't need to code everything  is already done for you . you just need to rebuild it according to `planning`

**Comment by sammou00** [comment]
Hello, 
1- Very good job, we all are very proud of you.

2- very nice `user-personas` 

3- very Nice `backlog`

4- very nice `development-strategy`

5- Where is the whole `html` and `css` ?

6- There are no comments in your `html` and `css`

**Reply by hadoan149** [reply]
Thank you so much 🚀 🚀 ! Note taken, will fix it now!

**Comment by bermarte** [comment]
Not sure about `styling the footer links to be to be gapped.` and `there is an image of Batman`.    
Please have a look to the text in the development strategy.
